mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/quota/CassandraPerUserMaxQuotaManager.java

```diff
@@ -133,31 +135,49 @@ public void removeDefaultMaxMessage() {
 
     @Override
     public Optional<QuotaSize> getMaxStorage(QuotaRoot quotaRoot) {
-        return perUserQuota.getMaxStorage(quotaRoot)
-            .map(Optional::of)
-            .orElseGet(Throwing.supplier(this::getDefaultMaxStorage).sneakyThrow());
+        Supplier<Optional<QuotaSize>> domainQuotaSupplier = Throwing.supplier(() -> quotaRoot.getDomain()
+            .flatMap(this::getDomainMaxStorage)).sneakyThrow();
+        Supplier<Optional<QuotaSize>> defaultDomainSupplier = Throwing.supplier(this::getDefaultMaxStorage).sneakyThrow();
```

Comment:
defaultDomain? You mean global?

Reply:
I guess the same vocabulary remarks applies in other max quota manager (memory, jpa)....

Reply:
As well as everywhere in the code base to make it consistant....